modules/javafx.web/src/test/java/test/javafx/scene/web/CSSTest.java

```diff
@@ -27,14 +27,24 @@
 
 import static javafx.concurrent.Worker.State.FAILED;
 import static javafx.concurrent.Worker.State.SUCCEEDED;
+
+import com.sun.webkit.WebPage;
+import com.sun.webkit.WebPageShim;
+
+import java.awt.*;
```

Comment:
As a best practice, we avoid Wildcard imports for packages (static imports of fields and methods within a class are sometimes fine for certain classes).